power_grid_model_c/power_grid_model/include/power_grid_model/common/statistics.hpp

```diff
@@ -0,0 +1,181 @@
+// SPDX-FileCopyrightText: Contributors to the Power Grid Model project <powergridmodel@lfenergy.org>
+//
+// SPDX-License-Identifier: MPL-2.0
+
+#pragma once
+
+#include "common.hpp"
+#include "three_phase_tensor.hpp"
+
+/**
+ * @file statistics.hpp
+ * @brief This file contains various structures and functions for handling statistical representations of
+ * randomly distributed variables(RDV) used in the Power Grid Model, like in the State Estimation algorithms to
+ * handle measurements.
+ *
+ * The structures provided in this file are used to represent measured values of sensors
+ * with different types of variances. These structures support both symmetric and asymmetric representations and
+ * provide conversion operators to transform between these representations.
+ *
+ * A Randomly distributed variable in PGM can have following characteristics:
+ *  - Uniform: Single Variance for all phases
+ *  - Independent: Unique Variance for each phase
+ *  - Real: The Real value without direction, eg. real axis: RealValue (* 1), imaginary axis: RealValue (* 1i).
+ *  - Complex: A combined complex value in `a + bi` notation.
+ *
+ * Based on these, we use combine variables in Polar/Decomposed forms:
+ * - Decomposed: Treat RDV individually as in cartesian co-ordinates with a separate variance for both real and
+ * complex component.
+ * - Polar: RDV is in polar co-ordinates, with magnitude and angle.
```

Comment:
This part is a bit hard to flow from the above, perhaps some inbetween liaison?
```suggestion
 * [liaison]
 * Based on these, we use combined variables in Decomposed/Polar forms:
 * - Decomposed: treat random variables individually as in cartesian co-ordinates with separated variances for both real and
 * imaginary part.
 * - Polar: random variables are in polar co-ordinates, with magnitudes and angles.
```